### PR TITLE
Added Backend Api for the tournaments

### PR DESCRIPTION
**Pull request by AayushSaini101**
Added Backend Api for the tournaments

### CODE REVIEW

File: controller/tournamentController.js [hunk 1]
```diff
@@ -57,4 +57,16 @@ function joinTournament(req, res) {
         }
     });
 }
-module.exports = { createNewTournament, viewAllTournamentDetails, joinTournament };
+
+//Function to get the tournament details using the tournament ID
+function getTournamentDetails(req, res) {
+    tournament.findOne({ tournamentId: req.body.tournamentId }, function(err, tournament) {
```
Comment (Aditya-Thakur) [inline controller/tournamentController.js:63]: Its a get method, so tournamentId can be accessed using params not body, please fix it

File: router/tournament.router.js [hunk 2]
```diff
@@ -18,9 +18,14 @@
...
~
~router.get("/join",(req,res)=>{
~    console.log("Joining the tournament");
     joinTournament(req,res);
 })
 
+router.get("/getTournament/:id",(req,res)=>{
+    console.log("Getting the tournament details");
+    getTournamentDetails(req,res);
+}
```
[the remark this answers is not in the document]
Comment (Aditya-Thakur) [reply router/tournament.router.js:29]: Also the “/join” route should be post route.

**Review by Aditya-Thakur — CHANGES_REQUESTED** [review]
Please fix

File: controller/tournamentController.js (hunk 1 above)
Comment (AayushSaini101) [reply controller/tournamentController.js]: @Aditya-Thakur  Done

**Pull request merged**